### ISSUE TITLE
dcc redesign: RangeSlider inputs should focus together

### ISSUE THREAD

**Issue by KoolADE85** [plotly/dash#3434]
When tabbing through the inputs of a RangeSlider, the tab order should be:
1. Left Input
2. Right Input
3. Slider Handles (in left-to-right order)

Currently, the Right Input is focused last.
